build/build.xml

```diff
@@ -11,11 +11,11 @@
 
 <project name="tn3270" xmlns:if="ant:if" xmlns:unless="ant:unless">
     <target name="deploy">
-      <mkdir dir="${deploy}/instance/ZLUX/pluginStorage/com.rs.mvd.tn3270/sessions"/>
+      <mkdir dir="${deploy}/instance/ZLUX/pluginStorage/org.zowe.terminal.tn3270/sessions"/>
```

Comment:
Should we provide a migration script? Or at least tell users what they need to do to clean up their config directories?